storage/txcache/score_test.go

```diff
@@ -1,76 +1,95 @@
 package txcache
 
 import (
+	"strconv"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 func TestDefaultScoreComputer_computeRawScore(t *testing.T) {
-	computer := newDefaultScoreComputer(100)
+	_, txFeeHelper := dummyParamsWithGasPrice(100 * oneBillion)
+	computer := newDefaultScoreComputer(txFeeHelper)
 
-	score := computer.computeRawScore(senderScoreParams{count: 14000, size: kBToBytes(100000), fee: toNanoERD(300), gas: 2500000000})
-	require.InDelta(t, float64(0.1789683371), score, delta)
+	// 50k moveGas, 100Bil minPrice -> normalizedFee 8940
+	score := computer.computeRawScore(senderScoreParams{count: 1, feeScore: 18000, gas: 100000})
+	assert.InDelta(t, float64(34.78839226910904), score, delta)
 
-	score = computer.computeRawScore(senderScoreParams{count: 19000, size: kBToBytes(3000), fee: toNanoERD(2300), gas: 19000000000})
-	require.InDelta(t, float64(0.2517997181), score, delta)
+	score = computer.computeRawScore(senderScoreParams{count: 1, feeScore: 1500000, gas: 10000000})
+	assert.InDelta(t, float64(19.61086955543323), score, delta)
 
-	score = computer.computeRawScore(senderScoreParams{count: 3, size: kBToBytes(2), fee: toNanoERD(0.04), gas: 400000})
-	require.InDelta(t, float64(5.795382396), score, delta)
+	score = computer.computeRawScore(senderScoreParams{count: 1, feeScore: 5000000, gas: 30000000})
+	assert.InDelta(t, float64(26.60960295569976), score, delta)
 
-	score = computer.computeRawScore(senderScoreParams{count: 1, size: kBToBytes(0.3), fee: toNanoERD(0.05), gas: 100000})
-	require.InDelta(t, float64(100), score, delta)
+	score = computer.computeRawScore(senderScoreParams{count: 2, feeScore: 36000, gas: 200000})
+	assert.InDelta(t, float64(23.12944684325884), score, delta)
+
+	score = computer.computeRawScore(senderScoreParams{count: 1000, feeScore: 18000000, gas: 100000000})
+	assert.InDelta(t, float64(3.9327419187495716), score, delta)
+
+	score = computer.computeRawScore(senderScoreParams{count: 10000, feeScore: 180000000, gas: 1000000000})
+	assert.InDelta(t, float64(3.000829279334849), score, delta)
 }
 
 func BenchmarkScoreComputer_computeRawScore(b *testing.B) {
-	computer := newDefaultScoreComputer(100)
+	_, txFeeHelper := dummyParams()
+	computer := newDefaultScoreComputer(txFeeHelper)
 
 	for i := 0; i < b.N; i++ {
 		for j := uint64(0); j < 10000000; j++ {
-			computer.computeRawScore(senderScoreParams{count: j, size: (j + 1) * 500, fee: toNanoERD(float64(0.011) * float64(j)), gas: 100000 * j})
+			computer.computeRawScore(senderScoreParams{count: j, feeScore: uint64(float64(8000) * float64(j)), gas: 100000 * j})
```

Comment:
Can we extract some numbers on how much did the performance improve?

Reply:
I got these numbers
previous:
```
goos: linux
goarch: amd64
pkg: github.com/ElrondNetwork/elrond-go/storage/txcache
BenchmarkScoreComputer_computeRawScore-4   	       1	1281265157 ns/op
PASS
```

and now:
```
goos: linux
goarch: amd64
pkg: github.com/ElrondNetwork/elrond-go/storage/txcache
BenchmarkScoreComputer_computeRawScore-4   	       2	 908826040 ns/op
PASS
```